GA-worksheet.md

```diff
@@ -25,45 +25,51 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag? The <link> tag lets you include CSS from an external file. 
 
-2. What is normalize.css?
+2. What is normalize.css? It's a CSS file that preserve the CSS through any broswer. 
 
-3. Why do we need to add a padding property to the `<ul>` tag?
+3. Why do we need to add a padding property to the `<ul>` tag? We need to add a padding property to the <ul> tag because
+it centers the items on the list. 
```

Comment:
In the dash tutorial, the `text-align: center' in the header is actually what makes it centered.
There is default padding on the left for unordered lists, so adding padding to all sides makes this particular ul centered.